src/invariant/OrderBookIsNotCrossed.h

```diff
@@ -0,0 +1,84 @@
+#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
+#pragma once
+
+// Copyright 2019 Stellar Development Foundation and contributors. Licensed
```

Comment:
Year should be 2021. This comment applies to the two other files you added as well.

Reply:
Done.